components/About/AboutDetails.styled.tsx

```diff
@@ -0,0 +1,116 @@
+import styled from "styled-components";
+
+export const ContentContainer = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 16px;
+  padding: 42px;
```

Comment:
you can use [styled component themes](https://styled-components.com/docs/advanced) here to create some variables for spacing/font-size/colors

or you can use [CSS variables](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties) that have the same idea, but it's kinda less effective once you're already using styled-components - they would work better if you were writing pure CSS